_dev/src/ts/pages/UpdatePageBackup.ts

```diff
@@ -1,13 +1,70 @@
+import api from '../api/RequestHandler';
+import ModalContainer from '../components/ModalContainer';
 import UpdatePage from './UpdatePage';
 
 export default class UpdatePageBackup extends UpdatePage {
   protected stepCode = 'backup';
-
-  constructor() {
-    super();
-  }
+  protected formFields: Element[] = [];
 
   public mount() {
     this.initStepper();
+    this.form.addEventListener('submit', this.onFormSubmit);
+    this.form.addEventListener('change', this.onInputChange);
+
+    document.getElementById('ua_container')?.addEventListener('click', this.onClick);
+    document
+      .getElementById(ModalContainer.containerId)
+      ?.addEventListener(ModalContainer.okEvent, (ev) => {
+        // We handle the backup confirmation modal as it is really basic
+        if ((ev.target as HTMLElement).id === 'modal-confirm-backup') {
+          api.post(this.form.dataset.routeToConfirmBackup!);
+        }
+        // The update confirmation modal gets its logic in a dedicated script
+      });
+  }
+
+  public beforeDestroy(): void {
+    this.form.removeEventListener('submit', this.onFormSubmit);
+    this.form.removeEventListener('change', this.onInputChange);
+  }
+
+  private get form(): HTMLFormElement {
+    const form = document.forms.namedItem('update-backup-page-form');
+    if (!form) {
+      throw new Error('Form not found');
+    }
+
+    ['routeToSave', 'routeToSubmitBackup', 'routeToSubmitUpdate', 'routeToConfirmBackup'].forEach(
+      (data) => {
+        if (!form.dataset[data]) {
+          throw new Error(`Missing data ${data} from form dataset.`);
+        }
+      }
+    );
+
+    return form;
   }
+
+  private readonly onClick = async (ev: Event) => {
+    if ((ev.target as HTMLElement).id === 'update-backup-page-skip-btn') {
+      const formData = new FormData();
```

Comment:
Wouldn't it be more interesting to send a skip Backup method? This would give better readability on the action and could lead to other things.

Reply:
As discussed, this is currently a hardcoded value at the moment. The addition of the backup process will allow this value to change.
The main information we want to show this name is the backup that should be done, not skipped. In consequence this variable is named to show the main objective.